glide.lock

```diff
@@ -36,16 +36,22 @@ imports:
   version: 5bf94b69c6b68ee1b541973bb8e1144db23a194b
 - name: github.com/shopspring/decimal
   version: 3c692774ac4c47c7a296ec96e553719dba1a68fc
+- name: github.com/xtophs/azure-sdk-for-go
```

Comment:
Why is this here?

Reply:
This should not be here, created PR
https://github.com/Azure/azure-sdk-for-go/pull/787